project/Build.scala

```diff
@@ -33,7 +31,7 @@ object SonatypeBuild extends Build {
 
   import xerial.sbt.Sonatype.SonatypeKeys._
 
-  lazy val buildSettings = Defaults.defaultSettings ++ releaseSettings ++ scriptedSettings ++ xerial.sbt.Sonatype.sonatypeSettings ++ Seq[Setting[_]](
```

Comment:
Defaults.defaultSettings is deprecated, automatically included by JvmPlugin autoplugin.